petl/io/xlsx.py

```diff
@@ -178,7 +183,10 @@ def appendxlsx(tbl, filename, sheet=None, write_header=False):
         else:
             ws = wb[str(sheet)]
         if write_header:
-            rows = tbl
+            it = iter(tbl)
```

Comment:
Code scanning: Variable name "it" doesn't conform to snake_case naming style

Reply:
That code is used in bcolz, db, text. Reused it for some consistency, wouldn't touch it now IMO

Reply:
- I agree, we can wait for #597